sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/session/SessionManager.java

```diff
@@ -80,7 +76,6 @@ public synchronized void init(HiveConf hiveConf) {
     }
     createBackgroundOperationPool();
     addService(operationManager);
-    super.init(hiveConf);
```

Comment:
hmm, I think we revert keep this line too.

Reply:
yes, we could keep super.init() here and remove initCompositeService in SparkSQLSessionManager.init(). thanks